packages/react-ui-components/src/stories/Components/TextField.stories.tsx

```diff
@@ -0,0 +1,67 @@
+import { ComponentStory } from "@storybook/react";
+import React from 'react';
+import { KvTextField } from "../../components";
+
+KvTextField.displayName = 'KvTextField'
+
+export default {
+	title: "Components/Text Fields",
+	component: 'kv-text-field',
+	argTypes: {
+		disabled: { type: 'boolean' },
+		required: { type: 'boolean' },
+		loading: { type: 'boolean' },
+		helpText: { type: "text" }
+	},
+	parameters: {
+		notes: require('../../../../ui-components/src/components/text-field/readme.md')
+	}
+};
+
+const TextFieldTemplate: ComponentStory<typeof KvTextField> = (args) => <KvTextField {...args} />;
+
+
+export const Default = TextFieldTemplate.bind({});
+Default.args = {
+    label: 'Default Text Field',
+	disabled: false,
+	required: false,
+	loading: false,
+	helpText: ''
+};
+
+export const Disabled = TextFieldTemplate.bind({});
+Disabled.args = {
```

Comment:
instead of typing the props all over again, use the spread operator with the Default args like so:
`Disabled.args = { ...Default.args, disabled: true }`

Apply this style in the other state variations of this story.